users/postgres/init.go

```diff
@@ -56,6 +56,11 @@ func migrateDB(db *sqlx.DB) error {
 						email	 VARCHAR(254) PRIMARY KEY,
 						password CHAR(60)	  NOT NULL
 					)`,
+					`CREATE TABLE IF NOT EXISTS tokens (
+						user_id    VARCHAR(254),
+						token	 VARCHAR(254),
+						FOREIGN KEY (user_id) REFERENCES users (email) ON DELETE CASCADE ON UPDATE CASCADE
+					)`,
```

Comment:
Please create a migration for this (just move `tokens` part into the `users_2`).